lr1110/lr1110/src_changed/smtc_hal_mcu.c

```diff
@@ -57,7 +59,7 @@ void hal_hex_to_bin( char *input, uint8_t *dst, int len )
 {
 	char tmp[3];
 	uint16_t length = strlen( input );
-	tmp[2] = NULL;
+	tmp[2] = '\0';
```

Comment:
Big fan of this change. I frequently tell my students 0, False, '\0', and NULL are separate things and should be used in semantically meaningful ways. /rant